core-rust/node-common/src/java/utils.rs

```diff
@@ -115,6 +115,21 @@ pub fn jni_sbor_coded_fallible_call<Args: ScryptoDecode, Response: ScryptoEncode
     .unwrap_or_else(std::ptr::null_mut)
 }
 
+/// A convenience method for the case when input should remain in the form of raw byte array
+/// (i.e. no SBOR decoding is needed). Output is still properly encoded with SBOR.
+pub fn jni_raw_sbor_fallible_call<Response: ScryptoEncode>(
```

Comment:
This is an SBOR encoded structure, and it is automatically SBOR-encode on the Java side. On the Rust side it has no use now and there is no sense decoding it because DB contains SBOR encoded struct anyway. There were two reasonable solutions 
- manually encode SBOR on the Java side, wrap it into array and when it comes back - manually unwrap and decode
- get direct access to a byte array before it is decoded on the Rust side and pass it to the relevant underlying method

The second approach looked much simpler to me.

Reply:
No longer an issue, and the corresponding helper is removed.